[gameplay]/mapfixes/server.lua

```diff
@@ -0,0 +1,28 @@
+for name, data in pairs(mapFixComponents) do
+    data.enabled = get(name) == "true"
+end
+
+local function handlePlayerResourceStart(res)
+    if res ~= resource then return end
+
+    triggerClientEvent(source, "mapfixes:client:loadAllComponents", source, mapFixComponents)
+end
+addEventHandler("onPlayerResourceStart", root, handlePlayerResourceStart)
+
+local function handleSettingChange(settingName)
+    settingName = settingName:gsub(getResourceName(resource)..("."), "")
+    local modifier = settingName:sub(1, 1)
+    if modifier == "*" or modifier == "#" or modifier == "@" then
+        settingName = settingName:sub(2)
+    end
+    local data = mapFixComponents[settingName]
+    if not data then return end
+    
+    local newValue = get(settingName)
+    data.enabled = newValue == "true"
+    
+    for _, player in pairs(getElementsByType("player")) do
+        triggerClientEvent(player, "mapfixes:client:togOneComponent", player, settingName, data.enabled)
```

Comment:
```suggestion
    triggerClientEvent(root, "mapfixes:client:togOneComponent", resourceRoot, settingName, data.enabled)
```

Single trigger for everyone instead of trigger * amount of players. Do not forget to change addEventHandler for resourceRoot

Reply:
Ok I'll make it a single trigger. But it's source player, look at the event handler in the client side. It's attached to localPlayer only due to propagate false.